src/libs/Network/index.ts

```diff
@@ -35,8 +30,8 @@ function post(command, data = {}, type = CONST.NETWORK.METHOD.POST, shouldUseSec
         // (e.g. any requests currently happening when the user logs out are cancelled)
         request.data = {
             ...data,
-            shouldRetry: lodashGet(data, 'shouldRetry', true),
-            canCancel: lodashGet(data, 'canCancel', true),
+            shouldRetry: data?.shouldRetry ?? true,
+            canCancel: data?.canCancel ?? true,
```

Comment:
No need for the `?`, no?

Reply:
Same https://github.com/Expensify/App/pull/28031#discussion_r1345579630